src/chapter1/04-control-flow.md

```diff
@@ -207,7 +223,10 @@ Uncaught TypeError: Cannot read properties of undefined (reading 'day')
 This makes sense since `mernTask.date` will result in `undefined` and you can't access a property on `undefined`. But let's say we would like to access the day and setting it to `undefined` if the `date` property is not present. Then we would need to do something like the following:
 
 ```javascript
-const day = mernTask.date !== null ? mernTask.date.day : undefined;
+const day = mernTask.date != null ? mernTask.date.day : undefined;
+
+// or without loose equality
+const day = typeof mernTask.date !== undefined && mernTask.date !== null ? mernTask.date.day : undefined;
 ```
```

Comment:
I hate both solutions and I don't think we should show both but it definitely needs to be fixed. 
Any ideas?

Reply:
We should definitely choose the second one, since we banned !=

Also the fact that the second solution is even uglier now is great, isn't it? This will highlight the use case for optional chaining even more.